addons/mrp/i18n/mrp.pot

```diff
@@ -80,6 +80,41 @@ msgstr ""
 msgid "%i work orders"
 msgstr ""
 
+#. module: mrp
+#: code:addons/mrp/data/mrp_data.xml:0
+msgid "Material Availability"
+msgstr ""
+
+#. module: mrp
+#: code:addons/mrp/data/mrp_data.xml:0
+msgid "Equipment Failure"
+msgstr ""
+
+#. module: mrp
+#: code:addons/mrp/data/mrp_data.xml:0
+msgid "Setup and Adjustments"
+msgstr ""
+
+#. module: mrp
+#: code:addons/mrp/data/mrp_data.xml:0
+msgid "Reduced Speed"
+msgstr ""
+
+#. module: mrp
+#: code:addons/mrp/data/mrp_data.xml:0
+msgid "Process Defect"
+msgstr ""
+
+#. module: mrp
+#: code:addons/mrp/data/mrp_data.xml:0
+msgid "Reduced Yield"
+msgstr ""
+
+#. module: mrp
+#: code:addons/mrp/data/mrp_data.xml:0
+msgid "Fully Productive Time"
```

Comment:
If for some reason the term isn't part of the pot file export, then simply manually adding the term into the pot file **won't** make them translated. I suspect you may have done that considering that you added these terms are used in a not translatable name field  in v16, which we fixed in saas-16.2 onwards: https://github.com/odoo/odoo/pull/113641